contrib/buildahimage/stable/Containerfile

```diff
@@ -11,7 +11,9 @@ FROM fedora:latest
 # Don't include container-selinux and remove
 # directories used by yum that are just taking
 # up space.
-RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install buildah fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
+RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install buildah fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
+
+COPY containers.conf /etc/containers/
```

Comment:
These are called from quay.io to build our images when the PR merge.  I'm not sure that we can use a file there that isn't a "Dockerfile".  In addition, I don't think the COPY will work unless we fully specify the file location.  I've not tried this, but could we do something like:

```
COPY https://github.com/containers/buildah/tree/master/contrib/buildahimage/stable/containers.conf /etc/containers
```

Reply:
Could we check to see if this works or not, and then do this in a separate PR if it blows up.  I hate to muck up the Dockerfiles files like this, if we don't have to.

Reply:
COPY won't work, but an ADD like this does.

```
ADD https://raw.githubusercontent.com/containers/buildah/master/contrib/buildahimage/stable/containers.conf /etc/containers/
```